source/Editor.js

```diff
@@ -685,6 +694,12 @@ proto._docWasChanged = function () {
     this.fireEvent( 'input' );
 };
 
+// Gets the size of the document in bytes.
+var documentSizeInBytes = function () {
+    // Javascript uses 2 bytes per character
+    return this._getHTML().length * 2;
```

Comment:
i'd rather this function not rely on the document being passed into it as a parameter. is there a performance concern?